src/gui/SearchWidget.cpp

```diff
@@ -43,6 +44,8 @@ SearchWidget::SearchWidget(QWidget* parent)
     m_searchTimer->setSingleShot(true);
     m_clearSearchTimer->setSingleShot(true);
 
+    new QShortcut(QKeySequence(Qt::CTRL + Qt::Key_J), this, SLOT(toggleHelpShortcut()));
```

Comment:
Just call ToggleHelp directly

Reply:
Initially I did. But the issue owner requested that the shortcut would work only if the search field was focused. Hence I added two if statements to take that into account.. I can remove them if that is not necessary though and call `toggleHelp()` directly.

Reply:
You need to apply the shortcut to the widget, right now it's an application level shortcut. Check the qt docs.

Reply:
Tried solving it this way: 
```cpp 
QShortcut *searchShortcut = new QShortcut(QKeySequence(Qt::CTRL + Qt::Key_J), this); 
searchShortcut->setContext(Qt::WidgetWithChildrenShortcut);
connect(searchShortcut, SIGNAL(activated()), this, SLOT(toggleHelp())); 
```
However, this only works once. After the popup appears, the window is not focused which results in the shortcut not working (toggling on works, toggling off does not). Am I missing something here? Otherwise I would suggest simply binding the shortcut to `SearchWidget` and not worry about the focus on the searchbar...?

Reply:
Made the necessary changes, works great